pfcpiface/pfcpiface.go

```diff
@@ -449,6 +473,27 @@ func handleSessionDeletionRequest(msg message.Message, addr net.Addr, sourceIP s
 
 	log.Println("Got a session deletion request from: ", addr)
 
+	/* retrieve sessionRecord */
+	sessItem := sessions[(sdreq.SEID() >> 2)]
+
+	/* create context */
+	ctx, cancel := context.WithTimeout(context.Background(), 1000*time.Millisecond)
+	defer cancel()
+	done := make(chan bool)
```

Comment:
where is done being used to wait for completion or context timeout? example `createEntries` and `deleteEntries`. See if we can modify and reuse of those functions for both sim and actual pfcp purposes. Otherwise they need not be defined on `type upf`

Reply:
This will be fixed in the next commit. Probably need to re-arrange the code a bit for this.